src/js/index.js

```diff
@@ -0,0 +1,114 @@
+let inputArray = []
+let input = ''
+
+const updateValue = (value) => {
+  const isLastValueOperation = Number.isNaN(Number(inputArray.at(-1)))
+
+  if (isLastValueOperation) {
+    input = ''
+  }
+
+  if (input.length === 3) {
+    alert('숫자는 세 자리까지만 입력 가능합니다!')
+    return
+  }
+
+  input += value
+
+  if (isLastValueOperation) {
+    inputArray[inputArray.length === 0 ? 0 : inputArray.length] = input
+  } else if (!isLastValueOperation) {
+    inputArray[inputArray.length - 1] = input
+  }
+
+  render()
+}
+
+const updateOperation = (operation) => {
+  if (!inputArray.at(-1) || Number.isNaN(Number(inputArray.at(-1)))) {
+    alert('숫자를 먼저 입력한 후 연산자를 입력해주세요!')
+    return
+  }
+
+  if (operation === '=') {
+    calculateInputArray()
+    return
+  }
+
+  const currentIndex = inputArray.length === 0 ? 0 : inputArray.length
+  inputArray[currentIndex] = operation
+
+  render()
+}
+
+const calculateInputArray = () => {
+  const [num1, operation, num2] = inputArray
+  const [number1, number2] = [+num1, +num2]
+
+  switch (true) {
+    case operation === '+':
+      render(number1 + number2)
+      break
+
+    case operation === '-':
+      render(number1 - number2)
+      break
+
+    case operation === 'X':
+      render(number1 * number2)
+      break
+
+    case operation === '/':
+      render(Number(number1 / number2).toFixed(0))
+      break
+  }
+}
+
+const resetInput = () => {
+  inputArray = []
+  input = ''
+
+  render()
+}
+
+const render = (calculateResult) => {
+  const total = document.querySelector('#total')
+  const expression = inputArray.length === 0 ? 0 : inputArray.join('')
```

Comment:
```suggestion
  const expression = Number(inputArray.join(''));
```
이렇게 표현할 수 있답니다!
![image](https://user-images.githubusercontent.com/18749057/178938866-9b51bf29-fbe3-4946-b0b2-cb357d47d6e9.png)